app/models/provider.rb

```diff
@@ -98,4 +98,19 @@ def unassigned_site_codes
   def can_add_more_sites?
     sites.size < Site::POSSIBLE_CODES.size
   end
+
+  def external_contact_info
+    enrichments.published.order(created_at: :desc).first
+      .attributes
+      .slice(
+        'address1',
```

Comment:
Can we add `website`, please?

FYI on `Provider` it's called `url`.

Reply:
Good call, you wanna reject it?

Reply:
Done @defong

Reply:
Done. You guys look like circling sharks looking for a reason to reject PRs. :) I'll worry about the fallback to `url` stuff later.

Reply:
He made me do it @misaka